app/editor/src/features/admin/ministers/MinisterList.tsx

```diff
@@ -16,14 +16,18 @@ const MinisterList: React.FC = () => {
   const [{ ministers }, api] = useMinisters();
 
   const [items, setItems] = React.useState<IMinisterModel[]>([]);
+  const [isLoading, setIsLoading] = React.useState(false);
```

Comment:
Take a look at the `ReportCard.tsx` component.  It used the `useApp()` hook state property `requests` to determine if it's loading.  Then you don't need to manage another state variable everywhere.